Content.Client/Audio/ContentAudioSystem.AmbientLoop.cs

```diff
@@ -0,0 +1,131 @@
+using System.Linq;
+using Content.Client.Gameplay;
+using Content.Shared.Audio;
+using Content.Shared.CCVar;
+using Content.Shared.Random;
+using Robust.Shared.Audio;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Player;
+using Robust.Shared.Random;
+using Robust.Shared.Utility;
+
+namespace Content.Client.Audio;
+
+public sealed partial class ContentAudioSystem
+{
+    private EntityUid? _ambientLoopStream;
+    private AmbientLoopPrototype? _loopProto;
+    private readonly TimeSpan _ambientLoopUpdateTime = TimeSpan.FromSeconds(5f);
+
+    private const float AmbientLoopFadeInTime = 5f;
+    private const float AmbientLoopFadeOutTime = 10f;
+    private TimeSpan _nextLoop;
+
+    private void InitializeAmbientLoop()
+    {
+        Subs.CVar(_configManager, CCVars.AmbientMusicVolume, AmbienceCVarChangedAmbientLoop, true);
+
+        _nextAudio = TimeSpan.Zero;
+    }
```

Comment:
There's a lot of bizarre code in here that's not documented and was copied from the existing ambient code that is extremely out of date.

e.g.
- Why is there a 5 second update loop timer.
- Why is the there a volume field here
- Why does it check for gameplay state instead of just subbing to state changes.
- I can even spot my variable naming being copied.

Reply:
I don't fully understand this outdated code to rewrite it. 

e.g.

1) we don't need to check and execute the code to change the background hum every frame, for optimization I think
2) where?
3) I follow the code principles of similar systems when I don't fully understand what's going on.
4) they're not just copied, they're reused, since it's a partial